doc/modules/ROOT/pages/extensions.adoc

```diff
@@ -58,24 +52,8 @@ hosted outside the nREPL GitHub organization.
 * https://clojure-emacs.github.io/sayid/[sayid]: A powerful tracing/debugging tool. It is a great alternative of CIDER-nREPL's basic tracing functionality.
 * https://github.com/vvvvalvalval/scope-capture-nrepl[scope-capture-nrepl]: A companion library to scope-capture, providing an nREPL middleware that lets you put your REPL in the context of an Execution Point.
 
-==== Deprecated
-
-This section lists middleware that were somewhat prominent in the
-past, but were replaced by alternatives down the road.
-
-* https://github.com/pallet/ritz/tree/develop/nrepl-middleware[nrepl-middleware],
-  part of https://github.com/pallet/ritz[ritz] that provides a
-  variety of nREPL middleware supporting various enhanced REPL
-  operations (including apropos, javadoc lookup, code completion, and
-  an alternative eval implementation). **(superseded by `cider-nrepl`)**
-* https://github.com/technomancy/javert[Javert] provides a basic
-  object inspector. **(superseded by `cider-nrepl`)**
-* https://github.com/thunknyc/nrepl-profile[nrepl-profile]: profiling middleware,
-which was eventually integrated into `cider-nrepl`.
-
-
 [NOTE]
 ====
-This list doesn't aim to be complete. You can find more 3rd-party middleware listed
+More 3rd-party extensions (including deprecated/abandoned ones) are listed
```

Comment:
Probably we can remove this section and the wiki page, as I had totally forgotten about them at this point. :D